### PR TITLE
Agenda for Dec 15 meeting

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

File: notes/meetings/2021-12/december-15.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+* ES to show results
+
+## Prioritisation of actions related to survey findings
+
+* TODO 
+
+## Creating a skills register
+
+* TODO
```
Comment (dominikwilkowski) [inline notes/meetings/2021-12/december-15.md]: We should create a couple groups and associate them with specific areas.
- A technical group that has their word for technical implementation details
- A design group that has their word in terms of design and UX
- An a11y group that has their word when it comes to a11y
- Perhaps a content group if we end up having enough

File: notes/meetings/2021-12/december-15.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+* TODO 
+
+## Creating a skills register
+
+* TODO 
+
+## Maintainers commitments
+
+* Min time spend
```
Comment (dominikwilkowski) [inline notes/meetings/2021-12/december-15.md]: To be part of any of the aforementioned groups you must spend x amount of time on this project.
We should commit to at least 4 meetings per year such as this one where we go through the RFCs and give the project a push. Implementation can then be assigned to a few people between those meetings.

File: notes/meetings/2021-12/december-15.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## RFC approvals
+
+* [Remove pancake](https://github.com/designsystemau/RFCs/pull/1)
+* [Simplify monorepo (DRAFT)](https://github.com/designsystemau/RFCs/pull/2)
+* [Separate framework entry points (DRAFT)](https://github.com/designsystemau/RFCs/pull/3)
+* [Docs site improvements](https://github.com/designsystemau/RFCs/pull/4)
+* [Create STANDARDISED-ACCESSIBILITY-TESTING.md](https://github.com/designsystemau/RFCs/pull/6)
+* [Update Grid 12](https://github.com/designsystemau/RFCs/pull/11)
+* [Distribute unminified and unprefixed CSS](https://github.com/designsystemau/RFCs/pull/16)
```
Comment (dominikwilkowski) [inline notes/meetings/2021-12/december-15.md:58]: Last part:

```suggestion
* [Distribute unminified and unprefixed CSS](https://github.com/designsystemau/RFCs/pull/16)

## Assigning people for implementation

 * [Remove pancake](https://github.com/designsystemau/RFCs/pull/1)
* [Simplify monorepo (DRAFT)](https://github.com/designsystemau/RFCs/pull/2)
* [Separate framework entry points (DRAFT)](https://github.com/designsystemau/RFCs/pull/3)
* [Docs site improvements](https://github.com/designsystemau/RFCs/pull/4)
* [Create STANDARDISED-ACCESSIBILITY-TESTING.md](https://github.com/designsystemau/RFCs/pull/6)
* [Update Grid 12](https://github.com/designsystemau/RFCs/pull/11)
* [Distribute unminified and unprefixed CSS](https://github.com/designsystemau/RFCs/pull/16)
```

Comment (elisechant) [reply notes/meetings/2021-12/december-15.md:58]: Re "## Assigning people for implementation", above comment you also say "Implementation can then be assigned to a few people between those meetings.", so which is better?

File: notes/meetings/2021-12/december-15.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+## Maintainers commitments
+
+* DW to facilitate 
+  * Min time spend
+  * 4 Steering Committee meetings / year
+
+## RFC approvals
+
+* [Remove pancake](https://github.com/designsystemau/RFCs/pull/1)
+* [Simplify monorepo (DRAFT)](https://github.com/designsystemau/RFCs/pull/2)
```
Comment (elisechant) [inline notes/meetings/2021-12/december-15.md]: @dominikwilkowski  this should be taken out of DRAFT or removed from this list

File: notes/meetings/2021-12/december-15.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* [Separate framework entry points (DRAFT)](https://github.com/designsystemau/RFCs/pull/3)
```
Comment (elisechant) [inline notes/meetings/2021-12/december-15.md]: this should be taken out of DRAFT or removed from this list

File: notes/meetings/2021-12/december-15.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+## Prioritisation of actions related to survey findings
+
+* MS to facilitate
+* Actions to be created in GitHub
+
+## Creating a skills register
+
+* DW to facilitate
+  * Creation of new Slack channels
+    * Technical group
```
Comment (elisechant) [inline notes/meetings/2021-12/december-15.md]: I think we should create these ahead of the meeting. 

If so, can we agree to naming conventions? putting forward:

#maintainers-technical
#maintainers-design
#maintainers-a11y
#maintainers-content

**Comment by elisechant** [comment]
- [ ] Include survey results @elisechant

**Pull request merged**